test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -44,21 +76,6 @@ describe('POST /group/:groupId/join', () => {
       });
     });
 
-    it('allows non-invited users to join public guilds', async () => {
-      let publicGuild = (await createAndPopulateGroup({
-        groupDetails: {
-          name: 'Test Guild',
-          type: 'guild',
-          privacy: 'public',
-        },
-      })).group;
-
-      let joiningUser = await generateUser();
-      await joiningUser.post(`/groups/${publicGuild._id}/join`);
-
-      await expect(joiningUser.get('/user')).to.eventually.have.property('guilds').and.to.include(publicGuild._id);
-    });
-
     context('User is invited', () => {
       it('allows invited user to join private guilds', async () => {
```

Comment:
There are tests for both parties and private guilds that they reject non-invited users.. am I missing something?